affinity/src/main/java/net/openhft/affinity/AffinityLock.java

```diff
@@ -400,6 +407,11 @@ public void bind(boolean wholeCore) {
 
     final boolean canReserve(boolean specified) {
 
+        if (isolationSettingsPreventUse()) {
+            LOGGER.debug("Cannot reserve CPU {} because it is not isolated by Chronicle Tune", cpuId);
```

Comment:
info?

Reply:
Thanks - that's updated to cover the two different use cases in which this might get called